scripts/start

```diff
@@ -41,9 +41,13 @@ if [[ ! -d "$UMBREL_ROOT" ]]; then
   exit 1
 fi
 
+# make sure NGINX_PORT andd NETWORK are set
+NGINX_PORT="${NGINX_PORT:-80}"
+NETWORK="${NETWORK:-mainnet}"
+
 # Configure Umbrel if it isn't already configured
 if [[ ! -f "${UMBREL_ROOT}/statuses/configured" ]]; then
-  NGINX_PORT=${NGINX_PORT:-80} NETWORK="${NETWORK:-mainnet}" "${UMBREL_ROOT}/scripts/configure"
+  "${UMBREL_ROOT}/scripts/configure"
```

Comment:
Please keep the code changes for this PR as low as possible because of Umbels 1 PR = 1 Feature/Bug Fix policy, this doesn't need to be changed to fix the issue.